lib/averages.js

```diff
@@ -0,0 +1,57 @@
+var pry = require('pryjs');
+
+var total = function(numbers) {
+  return numbers.reduce((sum, number) => sum + number);
+}
+
+var mean = function(numbers) {
+  const sum = total(numbers);
+  return sum/numbers.length;
+}
+
+var median = function(array) {
+  let len = array.length 
+  if(len % 2 != 0) {
+    let index = Math.ceil(array.length/2 - 1);
+    return array[index]
+  } else {
+    let topIndex = array.length/2;
+    let bottomIndex = topIndex - 1; 
+    return (array[topIndex] + array[bottomIndex])/2;
+  }
+}
+
+var mode = function(array) {
+  numCount = array.reduce(function(obj, num) {
```

Comment:
Make sure to use a var, let or const keyword when you declare a variable to avoid putting it into the global scope